frontend/src/citizen-frontend/calendar/ActionPickerModal.tsx

```diff
@@ -32,6 +36,15 @@ export default React.memo(function ActionPickerModal({
   return (
     <ModalBackground onClick={close}>
       <Container>
+        {isHolidayFormActive && (
+          <Action onClick={openHolidays} data-qa="calendar-action-holidays">
+            {i18n.calendar.newHoliday}
+            <IconBackground>
+              <FontAwesomeIcon icon={faTreePalm} size="1x" />
+            </IconBackground>
+          </Action>
+        )}
+        <Gap size="s" />
```

Comment:
Gap should be inside the conditional or we could remove the `Gap`s and just use flex gap.